js_modules/dagit/packages/core/src/execute/LaunchButton.tsx

```diff
@@ -63,7 +64,9 @@ export const LaunchButton = ({config, runCount}: LaunchButtonProps) => {
     runCount,
     disabled: config.disabled,
   });
+  const [telemetryRequest] = useTelemetryAction("Clicked LaunchPad Button");
```

Comment:
Might be worth destructuring in the hook itself, so you could just do something like:

```
const logButtonClick = useTelemetryAction("Clicked LaunchPad Button");
```

Reply:
Also, I know this is just a first pass, but I think we'd eventually want to standardize this into some sort of parseable action types instead of a human readable string like "Clicked LaunchPad Button".  That way, we don't have to do any sort of post-processing when we rename 'LaunchPad' to something else.